src/app/components/range/range.component.ts

```diff
@@ -0,0 +1,112 @@
+import { Component, OnInit, Input, Output, ViewChild, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'ash-range',
+  templateUrl: './range.component.html',
+  styleUrls: ['./range.component.scss']
+})
+export class RangeComponent implements OnInit {
+
+  @Input() min = 100;
+  @Input() max = 200;
+  @Input() parseFunction: any;
+  _values = [100, 200];
```

Comment:
Se refiere a los valores por defecto? En tal caso quizás mejor usar `[this.min, this.max]`.
Por otra parte, sólo como pregunta, el range es sólo doble? o simplemente es doble por defecto?

Reply:
Ya está corregido eso. También he metido código nuevo sobre referente a controlar el zindex de las bolas para cuando una esté encima de otra.

Respondiendo a tu pregunta, el range puede ser simple, doble, triple o cuantas bolas se quieran